worker/uniter/runner/jujuc/k8s-spec-get.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Canonical Ltd.
```

Comment:
This is renamed from pod-spec-get.go but somehow git thinks it was deleted and created so looks like a new file